blockchain/ethereum/contracts/erc20/token transferer/b20c5c97-32b4-4893-bd16-74b976c072db.json

```diff
@@ -0,0 +1,237 @@
+{
+  "name": "blockchain/ethereum/contracts/erc20/token transferer",
+  "interface": {
+    "inputs": {
+      "e44eab12-986b-4add-ad32-ad867762c4dc": {
+        "type": "string",
+        "name": "token address",
```

Comment:
Nuance thing, but the instance name should be rather verb. E.g execute contract, forward inputs etc.